doc/source/virtual-cluster/key-concepts.rst

```diff
@@ -0,0 +1,62 @@
+Key Concepts
+============
+
+.. _virtual-cluster-key-concepts:
+
+This page introduces key concepts for Ray virtual clusters:
+
+.. contents::
+    :local:
+
+Virtual Cluster
+---------------
+**Virtual cluster** provide logical resource isolation, enable multi-tenancy by allowing
+Ray cluster admins to easily manage multiple workloads on the same Ray cluster.
+
+A Virtual Cluster consists of a group of heterogeneous worker nodes or a group of
+child clusters :ref:`child cluster <virtual-cluster-divisible-cluster>`
+and :ref:`indivisible cluster <virtual-cluster-indivisible-cluster>`:
+
```

Comment:
A virtual cluster consists of a group of heterogeneous worker nodes and a collection of child clusters, including :ref:child clusters <virtual-cluster-divisible-cluster> and :ref:indivisible clusters <virtual-cluster-indivisible-cluster>: